.github/workflows/e2e.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     if: github.repository_owner == 'cloudflare' && (github.event_name != 'pull_request' || (github.event_name == 'pull_request' && contains(github.event.*.labels.*.name, 'e2e' )) || (github.event_name == 'pull_request' && github.head_ref == 'changeset-release/main'))
     name: "E2E Test"
     strategy:
+      fail-fast: false
```

Comment:
It would be cool if we could override this (perhaps via a label on the PR - is that possible?) so that we could run all the other tests that didn't fail to completion to see where we stand overall, rather than just getting the one failure result - especially if it was a flake.

Reply:
Maybe I'm misunderstanding what this config setting does, but I has thought that setting `fail-fast` to false would mean that all tests would always run to completion (even if one failed)

Reply:
Oops! Sorry. For some reason I had read that as `fail-fast: true`!!